### PR TITLE
feat: waiting room user journey ref: WSC-1011

### PR DESCRIPTION
**Pull request by lisaparma**

### CODE REVIEW

**Review by yukiLu6 — DISMISSED** [review]

File: src/store/slices/MeetingsStoreSlice.test.ts [hunk 1]
```diff
@@ -83,4 +86,91 @@ describe('Test components slice', () => {
...
+				result.current.addRoom(temporaryRoom);
+				result.current.addMeeting(scheduleMeeting);
+				result.current.setWaitingList(scheduleMeeting.id, ['userId0', 'userId1']);
+				result.current.setWaitingList(scheduleMeeting.id, ['userId2', 'userId3']);
+			});
+
+			// Check store data
+			const { waitingList } = result.current.meetings[temporaryRoom.id];
+			expect(waitingList).not.toBeNull();
+			expect(size(waitingList)).toBe(2);
```
Comment (CerDennis) [inline src/store/slices/MeetingsStoreSlice.test.ts:119]: Here check also the values are the new ones

**Comment by CerDennis** [comment]
image

User without conversations can't see the button to create scheduled rooms

**Review by CerDennis — CHANGES_REQUESTED** [review]
image
User without conversations can't see the button to create scheduled rooms

**Comment by CerDennis** [comment]
Waiting room media devices toggles are not working

https://github.com/zextras/carbonio-ws-collaboration-ui/assets/31632342/017a764c-d31d-437e-9030-0d1be6495cef

**Comment by CerDennis** [comment]
image
 I entered the waiting room with another user in another browser and in the wsc view of the moderator I see the snackbar that tells me someone is waiting to enter but I don't now in which room

**Comment by CerDennis** [comment]
> image I entered the waiting room with another user in another browser and in the wsc view of the moderator I see the snackbar that tells me someone is waiting to enter but I don't now in which room

For this one we can discuss further on with @AliceTincani if we can find an improvement

**Comment by CerDennis** [comment]
After reload of page I can't see the waiting list
- as moderator I enter the virtual room
- an user require to enter the meeting by the waiting room
- I receive the notification
- I leave the meeting
- I enter the meeting again
I expect to see the waiting list with the user inside but I don't see anything

File: src/meetings/views/AccessMeetingPageView.tsx [hunk 2]
```diff
@@ -4,22 +4,67 @@
...
+				});
+		}
+	}, [chatsBeNetworkStatus, goToInfoPage, meetingId]);
 
 	return (
-		<Container background={theme.palette.gray1.hover}>
-			{roomId && <AccessMeetingModal roomId={roomId} />}
+		<Container background="gray0">
+			{hasUserDirectAccess === true && <AccessMeetingModal roomId={roomId} />}
+			{hasUserDirectAccess === false && (
```
Comment (CerDennis) [inline src/meetings/views/AccessMeetingPageView.tsx:65]: Here if it's undefined we are not covering the case

File: src/meetings/views/InfoPage.tsx [hunk 3]
```diff
@@ -65,15 +65,38 @@ const InfoPage = (): ReactElement => {
...
 			descriptionLowerLabel = t(
 				'keepInTouchWithColleagues',
 				"Keep in touch with your colleagues or join your groups' meeting rooms"
 			);
 			break;
+		// TODO: add the correct translation
+		case PAGE_INFO_TYPE.MEETING_NOT_FOUND:
+		default: {
+			titleLabel = t('', 'Meeting not found');
+			centralLabel = t('', 'The meeting you are looking for does not exist');
```
Comment (CerDennis) [inline src/meetings/views/InfoPage.tsx]: Are missing keys on porpuse?

**Comment by lisaparma** [comment]
>  User without conversations can't see the button to create scheduled rooms

https://zextras.atlassian.net/browse/WSC-1250

**Comment by lisaparma** [comment]
> After reload of page I can't see the waiting list
> 
> * as moderator I enter the virtual room
> * an user require to enter the meeting by the waiting room
> * I receive the notification
> * I leave the meeting
> * I enter the meeting again
>   I expect to see the waiting list with the user inside but I don't see anything

https://zextras.atlassian.net/browse/WSC-1304

**Review by CerDennis — DISMISSED** [review]

**Review by yukiLu6 — DISMISSED** [review]

**Review by yukiLu6 — APPROVED** [review]

**Review by CerDennis — APPROVED** [review]

**Pull request merged**